### PR TITLE
Support better starting... status

### LINKED ISSUE #749: [DSG] Support better `starting...` feedback
When a given workbench is starting we have a somewhat hidden modal behind clicking on the `starting...` text.

- Give it a cursor hand icon (to denote it is clickable -- maybe we need additional hints to showcase it without hovering over it)
- Present an error icon when we know the modal is reporting an issue

cc @kywalker-rh is there something else we can do to bring attention to the `Starting...` text that it is clickable for more information? We could colour it blue like a link? Maybe an icon next to it?

### PR DESCRIPTION
**Pull request by DaoDaoNoCode**
## Description

Closes #749 

## How Has This Been Tested?

Try to create a notebook, check the new style of `starting...` text. If there are some errors, there will be an error icon next to the text, you can also hover it to get the brief problem.

## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: frontend/src/pages/projects/notebook/NotebookStatusPopover.tsx [hunk 1]
```diff
@@ -1,36 +1,39 @@
...
   stopNotebook,
+  labelText,
 }) => {
   const { notebook, runningPodUid, isStarting } = notebookState;
   const [isStartModalOpen, setStartModalOpen] = React.useState(false);
   const [unstableNotebookStatus, events] = useNotebookStatus(notebook, runningPodUid, isStarting);
   const notebookStatus = useDeepCompareMemoize(unstableNotebookStatus);
+  const [isPopoverVisible, setPopoverVisible] = React.useState(false);
+
+  const onClose = () => setPopoverVisible(false);
```
Comment (andrewballantyne) [inline frontend/src/pages/projects/notebook/NotebookStatusPopover.tsx:29]: Is this needed? why not inline it?

File: frontend/src/pages/projects/notebook/NotebookStatusToggle.tsx [hunk 2]
```diff
@@ -71,22 +70,10 @@ const NotebookStatusToggle: React.FC<NotebookStatusToggleProps> = ({ notebookSta
         </FlexItem>
         <FlexItem>
           <NotebookStatusPopover
```
Comment (andrewballantyne) [inline frontend/src/pages/projects/notebook/NotebookStatusToggle.tsx:72]: Thought we agreed to rename this to `NotebookStatusText`?

**Review by andrewballantyne — CHANGES_REQUESTED** [review]

File: frontend/src/pages/projects/notebook/NotebookStatusToggle.tsx (hunk 2 above)
Comment (DaoDaoNoCode) [reply frontend/src/pages/projects/notebook/NotebookStatusToggle.tsx:72]: Ah, I thought you were talking about the name of the scss file lol... will rename it.

**Review by andrewballantyne — APPROVED** [review]

File: frontend/src/pages/projects/notebook/NotebookStatusToggle.tsx (hunk 2 above)
Comment (andrewballantyne) [reply frontend/src/pages/projects/notebook/NotebookStatusToggle.tsx]: SCSS files ALWAYS match component filed names -- unless they are global things.

**Pull request merged**